_problems/post-subsidy-security.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: problem
+title:  "Post-subsidy Security"
+tags: consenus, mining, incentives, PoW
+status: "open"
+maintainer: VzxPLnHqr
+issue: 29
+---
+
+As the block subsidy drops, the network security budget may be inadequate if a robust fee market fails to develop. Two forms of the problem are overall low fees, and adequate-but-bursty fees. In both cases the network may fail to garner enough work to properly secure itself.
+
+A goal of this research is to define one or more *soft forks* which either permanently
+solve or, at minimum, stave off the problem, without sacrificing current community values.
+This means, for example, a hard-fork which introduces inflation will not be considered here.
+
+## Impact
+
+<!-- - Try not to repeat the description too much -->
+<!-- - Make it clear what the impact on the big picture of Bitcoin's evolution would be -->
+
+Ultimately a reduced security, if sustained for long enough, manifests as reduced confidence in the future of the network.
+Reduced confidence could trigger a further reduction in security, and a downward spiral ensues. If the 
+network cannot offer an attractive enough reward to those contributing to its security (miners), then this threat, while far
+off right now (2022), could become an existential threat. 
```

Comment:
One more idea that could be introduced. Hashrate can be considered as a security offer of which the demand in bitcoins react as an expression of users preferences over a choice of store of value (other cryptocurrencies, gold, real estate, ...) against a given threat and risks model. Higher the hashrate, higher would be the resources and coordination cost encumbered by any reorg entity.